packages/babel-traverse/src/path/context.js

```diff
@@ -51,17 +51,17 @@ export function _call(fns?: Array<Function>): boolean {
   return false;
 }
 
-export function isBlacklisted(): boolean {
-  const blacklist = this.opts.blacklist;
-  return blacklist && blacklist.indexOf(this.node.type) > -1;
+export function isDenylisted(): boolean {
+  const denylist = this.opts.denylist;
```

Comment:
Even if they are not documented, we should try to keep backwards compatibility.

```suggestion
  const denylist = this.opts.denylist ?? this.opts.blacklist;
```